artiq/firmware/libboard_artiq/hmc830_7043.rs

```diff
@@ -1,15 +1,3 @@
-/*
- * HMC830 config:
- * 100MHz input, 1.2GHz output
- * fvco = (refclk / r_divider) * n_divider
- * fout = fvco/2
- *
- * HMC7043 config:
```

Comment:
That part of the comment shouldn't be removed.

Reply:
Okay, I'm happy to revert that change.

My thinking was:

`* fvco = (refclk / r_divider) * n_divider`

that's information is now here, which seems like a better place for it: https://github.com/hartytp/artiq/blob/2621782f8c6bcf05f0e0ca85df42704922e5945b/artiq/firmware/libboard_artiq/hmc830_7043.rs#L113

```
- * fout = fvco/2
- *
- * HMC7043 config:
`

That information is now here https://github.com/hartytp/artiq/blob/2621782f8c6bcf05f0e0ca85df42704922e5945b/artiq/firmware/libboard_artiq/hmc830_7043.rs#L161-L182 and here https://github.com/hartytp/artiq/blob/2621782f8c6bcf05f0e0ca85df42704922e5945b/artiq/firmware/libboard_artiq/hmc830_7043.rs#L335

Personally, I don't like duplicating this kind of information in comments at the top of the file, since it's easy for people to forget to update them when changing the code.

Reply:
To check, which bit of the comment do you want back?

```
 * HMC7043 config:
 * dac clock: 600MHz (div=2)
 * fpga clock: 150MHz (div=8)
 * sysref clock: 9.375MHz (div=128)
```

or this bit

```
- * fvco = (refclk / r_divider) * n_divider
- * fout = fvco/2
```

Reply:
Sorry. I meant just the hmc7043 comment should not get lost.

Reply:
Okay, I'll reintroduce that if you'd prefer.

This still seems a bit redundant to me given https://github.com/hartytp/artiq/blob/ddb47524fb030c88ed4bd82f8dfdbd602ffd7366/artiq/firmware/libboard_artiq/hmc830_7043.rs#L159-L180 What about I just add the frequencies to the comments where the divider values are defined?

Reply:
done